### ISSUE TITLE
typing.io.IO not supported

### ISSUE THREAD

**Issue by dhermes** [jonparrott/sphinx-docstring-typing#4]
Using

```
       stream (IO[bytes]): The stream (i.e. file-like object) that will
           be uploaded.
```

does not produce a link to the `IO` type. It's likely due to the layout of the [typing doc][1].

[1]: https://docs.python.org/3.6/library/typing.html#typing.io

**Comment by theacodes** [comment]
Yeah, we probably just need to hoist those into https://github.com/jonparrott/sphinx-docstring-typing/blob/master/sphinx_docstring_typing/__init__.py#L19
